wp-content/plugins/core/src/Blocks/Block_Base.php

```diff
@@ -97,17 +83,21 @@ public function enqueue_core_block_editor_front_style(): void {
 			return;
 		}
 
+		wp_add_inline_style( $handle, file_get_contents( $src_path ) );
```

Comment:
@ChrisMKindred or @JMRhodes Just calling this `file_get_contents()` out. I know we had some trouble with WPVIP's file system and want to make sure this is both functional and acceptable from a performance standpoint. This based on [a WP Dev blog post](https://make.wordpress.org/core/2021/07/01/block-styles-loading-enhancements-in-wordpress-5-8/) and is inlining the css for core blocks is the ideal approach here for FE.

Reply:
@dpellenwood yes I would stick with `file_get_contents()`. 

VIP just recommends `wpcom_vip_file_get_contents()` as a caching wrapper. But that function is only available in the VIP environments. Even on local development, we don't have access to that function, so it's difficult to test.